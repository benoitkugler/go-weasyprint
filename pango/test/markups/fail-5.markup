<span strikethrough_color="rot">Text</span>
