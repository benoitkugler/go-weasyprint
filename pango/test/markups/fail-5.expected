ERROR: value of 'strikethrough_color' attribute on <span> tag should be a color specification, not 'rot'